### PR TITLE
added github from scratch. added philipp to authors. refreshed dianas description

### PR DESCRIPTION
**Pull request by fabiansch**

### CODE REVIEW

File: _posts/2013-12-03-githubfromscratch-hamburg-december.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+category:
+tags:
+  - github
+  - workshop
+  - hamburg
+  - beginners
+authors:
+  - diana
+  - philipp
+image: /assets/content/2013-12-03-githubfromscratch-hamburg-december/0.jpeg
```
Comment (gnunicorn) [inline _posts/2013-12-03-githubfromscratch-hamburg-december.md]: I can't find this or any of the other mentioned pictures. Did you forget to add them?

File: _posts/2013-12-03-githubfromscratch-hamburg-december.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+published: true
+layout: post
+title: "GitHub from Scratch - „I fork you and you fork me“"
+teaser: "OTS Hamburg has created a new workshop: Github from Scratch"
```
Comment (gnunicorn) [inline _posts/2013-12-03-githubfromscratch-hamburg-december.md]: This is the snippet we show on the index/overview page. I'd write a little more here to get people on the page wanting to read the article. In doubt, we usually just copy-paste the first paragraph here.

File: _posts/2013-12-03-githubfromscratch-hamburg-december.md (hunk 1 above)
Comment (fabiansch) [inline _posts/2013-12-03-githubfromscratch-hamburg-december.md]: oh yes. Sorry. Give me a few minutes ;)

Am 04.12.2013 um 10:59 schrieb Benjamin Kampmann:

**Comment by gnunicorn** [comment]
@fabiansch no reason to close. if you just push to the same branch all future changes become part of the same pull request.

**Comment by gnunicorn** [comment]
@fabiansch if you'd now add a bit more of a teaser, I'm happy to merge it.

File: _posts/2013-12-03-githubfromscratch-hamburg-december.md (hunk 2 above)
Comment (fabiansch) [inline _posts/2013-12-03-githubfromscratch-hamburg-december.md]: ah super. Okay :)

**Comment by fabiansch** [comment]
Done :)

**Pull request merged**